README.md

```diff
@@ -91,26 +91,39 @@ mvn spring-boot:stop
 # Make sure you are in the root of the repo
 # Run the command below to stop the previous instance and free up port 4120:
 
-mvn spring-boot:stop 
+mvn spring-boot:stop
 
 # You can also hit Ctrl-C from the shell window
 
-# Build the image and run the container using Docker 
+# Build the image and run the container using Docker
 
 # make sure you are in the root of the repo
 
 # this will use Azure CLI cached credentials
 
 # build the image
 
-docker build -t helium-dev .
+# IMPORTANT: In the following steps, you will be using the Dockerfile-Dev to build a developer
+# image, and thus, you MUST make sure that you change the UID of the 'helium' user set via the
+# 'useradd -u <uid>' in the Dockerfile-Dev to the one that is assigned to you on your host.
+# You may use the 'id' command to figure out your user id.
+
+docker build . -t helium-dev -f Dockerfile-Dev
```

Comment:
need to explain multi-stage build and how to build the helium-dev-base as a permanent image (so that you don't have to rebuild it when you run docker system prune)

Reply:
Used the language from C#